### PR TITLE
Assignment 4 @VeraKorchemnaya @AndrewKorchemniy

### PR DESCRIPTION
**Pull request by AndrewKorchemniy**
### Deployed to: [red-sand-0fa51601e.3](https://red-sand-0fa51601e.3.azurestaticapps.net/)

### CODE REVIEW

**Comment by TannerStephenson** [comment]
Will review

**Review by TannerStephenson — COMMENTED** [review]
Make all pages responsive ❌✅?
1080p desktop ✅
iPad Air ✅

Samsung Galaxy S20 Ultra Super Mega Excellent ❌✅?
The app bar menus are unreadable on it. Just partial words with '...'.

IPhone SE ❌✅?
Same issue as the samsung but they're even more cut off.

Create a improved landing page and site theme (Background, make it look attractive to play) ✅
Theme should extend all pages ✅
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
_This is completely optional and just an opinion:_ While you indicate which page you are on with the button, it's slightly confusing at first whether you need to click on the button to be brought to that page or if you're already there.

Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit) ❌

I really like your pages theme. The gradient switching is really cool. I think you just have to make the menus a little more readable on different devices and get the Stats page implemented and you should be good to go! Great job.

_Edit: After Grant went over the requirements in class, I think how I graded the responsive issues may be incorrect. I added back green check marks since everything else looks great._

File: wordle-web/src/App.vue [hunk 1]
```diff
@@ -27,18 +64,25 @@
...
+                <RouterLink to="/leaderboard">Leaderboard</RouterLink>
+              </v-list-item-title>
             </v-list-item>
           </v-list>
         </v-menu>
       </template>
     </v-app-bar>
 
-    <v-main>
+    <v-main class="animated-background">
```
Comment (mmwoodfo) [inline wordle-web/src/App.vue:85]: Very cool background!

**Review by mmwoodfo — COMMENTED** [review]
Good job - although I think there is supposed to be another page where I can play past games?

**Comment by mmwoodfo** [comment]
> Good job - although I think there is supposed to be another page where I can play past games?

Whoops! Nevermind. Found your page. :)

**Pull request closed without merging**